.github/workflows/rust.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: "Rust CI"
+on:
+  workflow_call:
+
+jobs:
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Prepare dependencies
+        run: |
+          rustup update stable
```

Comment:
Where does it make sure `rustup` is available? `runs-on: ubuntu-latest` does not imply rustup is present (to me)

Reply:
https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

They ship everything under the sun in terms of development tooling